packages/wast-parser/src/grammar.js

```diff
@@ -419,6 +417,8 @@ export function parse(tokensList: Array<Object>, source: string): Program {
         const fnParams = [];
         const fnResult = [];
 
+        let fnName = t.identifier(getUniqueName("func"));
```

Comment:
this change was not necessarily needed to make tests pass, but this does mean we have the same generated names when parsing wat

Reply:
Following https://github.com/xtuc/webassemblyjs/pull/411/files#r202406723 we could add our own func increment, or maybe we already have this information?